### ISSUE TITLE
Evaluate replacing "force game focus" with mouse trapping

### ISSUE THREAD

**Issue by GerRudi** [GerRudi/SimpleR-RotMG#5]
Instead of using force focus, rather use something like: 
````		
If WinActive($hWnd) Then
	$aCoords=WinGetPos($hWnd)
	_MouseTrap($aCoords[0]+10, $aCoords[1]+10, $aCoords[0] + $aCoords[2]-10, $aCoords[1] + $aCoords[3]-10)
Else
	_MouseTrap()
EndIf
````

Advantage: Actually avoid clicking out of the window and even avoid accidental resizing.

User can still "Tab out" or get out of the game window using Windows Key

To check:
Performance impact
Create Hotkey to disable/enable
Replace force focus
Check if coordinates are correct on older systems like windows 7

**Comment by GerRudi** [comment]
fixed with 062a900f6a09d62bde12859775b752ee32976f33

**Comment by GerRudi** [comment]
Implemented with #17
